package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "importer",
+  "version": "1.0.0",
+  "description": "",
```

Comment:
perhaps add a description and a repo property to this

Reply:
That's fair, it's done